### PR TITLE
Fix error configuration for cluster secret

### PR DESCRIPTION
**Pull request by godliness**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):

Bug Fix

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

fix official website documents which introduced cluster secret feature

Detailed description / Documentation draft:

distributed.md on official website is error for cluster secret feature.

"\<secret\>\</secret\>" should under "\<logs\>" not "\<shard\>"

### CODE REVIEW

**Comment by godliness** [comment]
@nikitamikhaylov

Seems like **Marker check** is pending.

File: docs/en/engines/table-engines/special/distributed.md [hunk 1]
```diff
@@ -61,19 +61,18 @@ Clusters are set like this:
...
 <remote_servers>
     <logs>
-        <shard>
-            <!-- Inter-server per-cluster secret for Distributed queries
-                 default: no secret (no authentication will be performed)
-
-                 If set, then Distributed queries will be validated on shards, so at least:
-                 - such cluster should exist on the shard,
-                 - such cluster should have the same secret.
+        <!-- Inter-server per-cluster secret for Distributed queries
```
Comment (alesapin) [inline docs/en/engines/table-engines/special/distributed.md:64]: Seems like the previous version was correct. We have such examples in tests https://github.com/ClickHouse/ClickHouse/blob/persistent_nukeeper_snapshot_storage/tests/integration/test_distributed_inter_server_secret/configs/remote_servers.xml#L15.

Comment (godliness) [reply docs/en/engines/table-engines/special/distributed.md:64]: @alesapin 

Alright, do we need fix this https://clickhouse.tech/docs/en/engines/table-engines/special/distributed/   in official website for distributed engine.

Comment (alesapin) [reply docs/en/engines/table-engines/special/distributed.md:64]: Both feature and documentation look correct. `<secret>` tag is a part of shard definition. So I don't understand where is the error?

Comment (godliness) [reply docs/en/engines/table-engines/special/distributed.md:64]: @alesapin  

https://github.com/ClickHouse/ClickHouse/blob/master/src/Interpreters/Cluster.cpp#L307

It's under cluster and all the replicas will share this secret.

Comment (alesapin) [reply docs/en/engines/table-engines/special/distributed.md:64]: Yes, you right. Sorry for the misunderstanding.

**Pull request merged**